sigh/lib/assets/resign.sh

```diff
@@ -380,6 +380,27 @@ function add_provision {
     add_provision_for_bundle_id "$PROVISION" "$BUNDLE_ID"
 }
 
+# codesign app and frameworks
+# codesign_bundle $BUNDLE_PATH (optional $ENTITLEMENTS)
+function codesign_bundle {
+  local BUNDLE_PATH=$1
+  local LOCAL_ENTITLEMENTS="$2"
+  
+  log "Resigning '$BUNDLE_PATH'"
+  log "using certificate: '$CERTIFICATE'"
+  
+  if [[ -n "$LOCAL_ENTITLEMENTS" ]]; then
+    log "and entitlements: $LOCAL_ENTITLEMENTS"
+    cp -f "$LOCAL_ENTITLEMENTS" "$BUNDLE_PATH/archived-expanded-entitlements.xcent"
```

Comment:
We don't need to copy `xcent` files into the bundle anymore. They're ignored because code sign embeds them into the app binary.